rero_ils/modules/loans/api.py

```diff
@@ -775,6 +807,29 @@ def is_overdue_loan(loan):
     return now > due_date + timedelta(days=days_after)
 
 
+def loan_has_open_events(loan_pid=None):
+    """Check if a loan has open patron transactions.
+
+    Loan has no open_events if the he has no related patron transaction with
+    the status open.
+
+    :return True|False.
+    """
+    notifications = []
+    search = NotificationsSearch().filter(
+        'term', loan__pid=loan_pid).source().scan()
```

Comment:
add 'pid' into `.source` to return only pid value ==> lighter ES call :: `.source(['pid']).scan()`